application/controllers/PrintanswersController.php

```diff
@@ -185,6 +185,15 @@ public function actionView($surveyid, $printableexport = false)
 
             $quexmlpdf = new quexmlpdf();
 
+            //apply settings stored at last output
```

Comment:
Code duplication? Looks like the same snippet as below?

Reply:
Yes, could be a routine of quexml?
`applyGlobalSettings()` ?

Reply:
thanks for the suggestion I've done that